releasenotes/notes/deprecate-distribution-learners-196741e53a8cecc5.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The `qiskit_machine_learning.algorithms.distribution_learners` package is deprecated. There's no
```

Comment:
I think we normally state, after noting its deprecated something like - and will be removed in a future version.

Reply:
Updated the reno, added "and will be removed no sooner than 3 months after the release".